src/package/midlrt/Microsoft.Windows.MidlRT.props

```diff
@@ -0,0 +1,42 @@
+<!--
+***********************************************************************************************
+Copyright (C) Microsoft Corporation.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <PropertyGroup>
+        <!-- Only do this for MSBuild versions below 16.0
+             as it is since done automatically, see https://github.com/microsoft/msbuild/pull/3605-->
+        <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)'  &lt;= '15'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+        
+        <UsingMidlRTPackage>true</UsingMidlRTPackage>
+    </PropertyGroup>
+
+     <PropertyGroup>
```

Comment:
I doubt any of these are needed

Reply:
I like having the `UsingFoo` properties just because they can be used by consuming projects.

Reply:
how many of these are needed (were some copied from cppwinrt?):

```
        <PreferredToolArchitecture>x64</PreferredToolArchitecture>
        <CanReferenceWinRT>true</CanReferenceWinRT>
        <IsNativeLanguage>true</IsNativeLanguage>
        <!-- This causes VS to add the platform WinMD references for the target SDK -->
        <WinMDAssembly>true</WinMDAssembly>
```

Reply:
`CanReferenceWinRT` is what VS uses to make sure your project can reference a .winmd - seems important to have. You could argue this would be set by projection packages - but it doesn't hurt

`WinMDAssembly`  is what VS uses to know this project produces a winmd - seems reasonable

`PreferredToolArchitecture` - we use preferred tool architecture in this nuget package

looks like we don't need  `IsNativeLanguage`